File: PSW-projekat/PZL-Angular/src/app/modules/hospital/model/doctorsCouncilDto.ts
import { DoctorDto } from "./doctorDto";

export class DoctorsCouncilDto {
    
    id: number = 0;
    topic: string = '';
    start: Date;
    duration: number = 0;
    doctors: DoctorDto[];

    public constructor(id: any, topic:any, start: any, duration:any, doctors: any) {
        this.id = id;
        this.topic = topic;
        this.start = start;
        this.duration = duration;
        this.doctors = doctors;
    }
}

File: PSW-projekat/hospital-be-main/src/HospitalLibrary/Core/DTOs/DoctorsCouncilDto.cs
using HospitalLibrary.Core.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HospitalLibrary.Core.DTOs
{
    public class DoctorsCouncilDto
    {
        public int Id { get; set; }
        public String Topic { get; set; }
        public DateTime Start { get; set; }
        public float Duration { get; set; }
        public virtual ICollection<DoctorDto> Doctors { get; set; }


        public DoctorsCouncilDto() { }

        public DoctorsCouncilDto(int id, string topic, DateTime start, float duration, ICollection<DoctorDto> doctors)
        {
            Id = id;
            Topic = topic;
            Start = start;
            Duration = duration;
            Doctors = doctors;
        }
    }

}
